### ISSUE TITLE
[Quest] Flight of the Wintergarde Defender

### ISSUE THREAD

**Issue by cathyr1** [WoWManiaUK/Redemption#1686]
Hello. Please check this quest http://www.wow-mania.com/armory/?quest=12237. Maybe it can be completed but it requires much more faster reaction that it shoud. My experience is awful. All NPCs dissapeared after 2 or 3 seconds so its nearly impossible to pick them. They should not dissapear or after much more longer time.

**Comment by cathyr1** [comment]
Ah, found another issue - No. #1676, currently marked as closed. So  just pls let us know when this will be fixed.

**Comment by fvicaria** [comment]
Sorry. It works as is. It has been fixed yesterday.

**Comment by cathyr1** [comment]
It can not work as it is now. Just try it. Before you can cast grab or during its cast, npc simply dissapears. You can spend there hour and grab 1 if you have luck and need 10 in total. It is soo annoying...

**Comment by fvicaria** [comment]
I have just tried in redemption this second. It worked fine.

**Comment by cathyr1** [comment]
I now achieved to save one but it is still bad. They dissapear too fast, some of them even when already on the gryphon. But if it works fine for you, is there some trick how to do it better?

**Comment by fvicaria** [comment]
I will re-check. There may be something wrong with them after they are picked for the first time and respawn.

**Comment by cathyr1** [comment]
I can confirm. I´ve grabbed him nearly in the 1st second (waiting on the spot until respawn) and then he dissapeared on the gryphon like 2-3 seconds after he sat down there.

**Comment by fvicaria** [comment]
I found the issue. They work fine the first time. After they respawn they have the problem you mentioned.

**Comment by Hulkster-WowMania** [comment]
I will rest this tonight when I get a chance if dev is still up.
